mlir/test/e2e/lit.local.cfg

```diff
@@ -0,0 +1,7 @@
+import importlib
+import shutil
+
+# Test file current forces python3.7, because it needs >=3.7.
+# Test requires tensorflow (which should be a recent tf-nightly).
+if not shutil.which('python3.7') or not importlib.util.find_spec('tensorflow'):
```

Comment:
Let's not have a hard dependency on 3.7 - check the python version this configuration is being run in instead